src/main/java/com/google/sps/datastore/GetDatastore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class GetDatastore {
+
+    public ArrayList<PCAPdata> dataTable = new ArrayList<PCAPdata>();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  public GetDatastore() 
+  {
+      
+  }
+
+
+  //get all properties in the entity "data" 
+  public void getAll(){
```

Comment:
a method something like getALL should return something. So just return the arraylist instead of populating it.

Reply:
Probably you wont need this method since the client can just call getEntity(PCAPInterface.DATASTORE_NAME);

Reply:
Method was removed